public/app/features/explore/Explore.test.tsx

```diff
@@ -289,7 +289,7 @@ describe('Explore', () => {
       await screen.findByTestId(selectors.components.DataSourcePicker.container);
 
       const addQueryButton = screen.getByRole('button', { name: /Add query$/i });
-      const addFromLibraryButton = screen.getByRole('button', { name: /Add query from library/i });
+      const addFromLibraryButton = screen.getByRole('button', { name: /Add from saved queries/i });
```

Comment:
@nmarrs 

> For all instances of "add saved query" should we maybe consider different wording here as this could mean that user is "adding" a new saved query vs using an existing one
> 
> So maybe "use saved query" would make more sense in all of these cases?

I decided to go with the "add from saved queries", the reasoning behind is because in this UI, the user is adding a new query into the query editor rows

<img width="2430" height="701" alt="Image" src="https://github.com/user-attachments/assets/dc358160-34b1-40f4-9771-6d4789e775ec" />